pkgs/applications/networking/listadmin/default.nix

```
@@ -0,0 +1,51 @@
+{ lib, stdenvNoCC, fetchurl, makeWrapper, perl, installShellFiles }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "listadmin";
+  version = "2.73";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/listadmin/${version}/listadmin-${version}.tar.gz";
+    sha256 = "00333d65ygdbm1hqr4yp2j8vh1cgh3hyfm7iy9y1alf0p0f6aqac";
+  };
+
+  buildInputs = [ perl ];
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  # There is a Makefile, but we don’t need it, and it prints errors
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    install -m 755 listadmin.pl $out/bin/listadmin
+    installManPage listadmin.1
+
+    wrapProgram $out/bin/listadmin \
+      --prefix PERL5LIB : "${with perl.pkgs; makePerlPath [
+        HTMLParser LWP LWPUserAgent MIMEBase64 HTMLTagset
+        HTTPMessage URI HTTPDate TryTiny TextReform
+        IOSocketSSL NetSSLeay NetINET6Glue EncodeLocale
+        LWPProtocolhttps NetHTTP
+        ]}"
```

`makeFullPerlPath` will also recursively resolve dependencies.

```suggestion
      --prefix PERL5LIB : "${with perl.pkgs; makeFullPerlPath [
        TextReform NetINET6Glue LWPProtocolhttps
        ]}"
```

---

Thanks, applied!